README.ja.md

```
@@ -0,0 +1,605 @@
+[![npm downloads](https://img.shields.io/npm/dm/mulmoclaude.svg)](https://www.npmjs.com/package/mulmoclaude)
+[![License: MIT](https://img.shields.io/npm/l/mulmoclaude.svg)](LICENSE)
+[![CI](https://github.com/receptron/mulmoclaude/actions/workflows/pull_request.yaml/badge.svg)](https://github.com/receptron/mulmoclaude/actions/workflows/pull_request.yaml)
+[![GitHub stars](https://img.shields.io/github/stars/receptron/mulmoclaude.svg?style=social)](https://github.com/receptron/mulmoclaude/stargazers)
+
+[English](README.md) · **日本語** · [简体中文](README.zh.md) · [한국어](README.ko.md) · [Español](README.es.md) · [Português (BR)](README.pt-BR.md) · [Français](README.fr.md) · [Deutsch](README.de.md)
+
+Claude Code との GUI チャット — さらに長期記憶、ビジュアルツール、メッセージングアプリ連携を搭載。
+
+Claude Code とチャットすると、単なるテキストだけでなく **インタラクティブなビジュアル出力** が得られます: ドキュメント、スプレッドシート、マインドマップ、チャート、画像、フォーム、3D シーンなど。組み込みの個人 Wiki により、Claude には会話のたびに成長する **永続的な知識** が備わります。
+
+**どこからでもアクセス**: Telegram、Slack、LINE、Discord、または [他の 10 個のメッセージングアプリ](#messaging-bridges) を接続して、スマートフォンから AI エージェントと会話できます。
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)